app/views/non_hmis_clients/assessments/_pathways_version_three_js.haml

```diff
@@ -27,17 +27,27 @@
     var warehouse_unsheltered_selector = 'input[name*="[homeless_nights_unsheltered]"]';
     var extra_sheltered_selector = 'input[name*="[additional_homeless_nights_sheltered]"]';
     var extra_unsheltered_selector = 'input[name*="[additional_homeless_nights_unsheltered]"]';
-    var total_selector = 'input[name*="[total_days_homeless_in_the_last_three_years]"]'
+    var total_selector = 'input[name*="[total_days_homeless_in_the_last_three_years]"]';
+    var additional_days_verified_selector = 'input[name*="[self_reported_days_verified]"]'
+    var additional_days_verified_checkbox = $('input.pretty_boolean[name*="[self_reported_days_verified]"]')[0];
+
+    var calculate_total_days = function(e) {
```

Comment:
@eanders Again, not sure about the logic here.

Reply:
Looks correct to me